provisioning_client/tests/prov_transport_amqp_ws_client_ut/prov_transport_amqp_ws_client_ut.c

```diff
@@ -416,10 +418,10 @@ BEGIN_TEST_SUITE(prov_transport_amqp_ws_client_ut)
     TEST_FUNCTION(prov_transport_amqp_register_device_succeed)
     {
         //arrange
-        STRICT_EXPECTED_CALL(prov_transport_common_amqp_register_device(TEST_DPS_HANDLE, on_transport_json_parse, NULL));
+        STRICT_EXPECTED_CALL(prov_transport_common_amqp_register_device(TEST_DPS_HANDLE, on_transport_json_parse, on_transport_create_json_payload, NULL));
 
         //act
-        int result = prov_amqp_ws_transport_register_device(TEST_DPS_HANDLE, on_transport_json_parse, NULL);
+        int result = prov_amqp_ws_transport_register_device(TEST_DPS_HANDLE, on_transport_json_parse, on_transport_create_json_payload, NULL);
```

Comment:
Looks like it is missing a negative test when create_json_payload callback is NULL